src/RealtimeEffectPanel.cpp

```diff
@@ -47,13 +47,79 @@
 #include "ListNavigationEnabled.h"
 #include "ListNavigationPanel.h"
 #include "MovableControl.h"
+#include "menus/MenuHelper.h"
+#include "Menus.h"
 
 #if wxUSE_ACCESSIBILITY
 #include "widgets/WindowAccessible.h"
 #endif
 
 namespace
 {
+   
+   class RealtimeEffectsMenuVisitor final : public MenuVisitor
```

Comment:
I am glad to see you figured out the usage of MenuVisitor!  I hope it did not seem unnecessarily difficult.  If you gained some more understanding of Registry.h and Registry.cpp that is good -- they need some re-work which will need careful review by someone.

Reply:
Had to spend some time to figure out how it works, yes :)